docker-compose.yml

```diff
@@ -75,6 +80,9 @@ services:
             RQ_ASYNC: 1
             DATABASE_URL: postgres://hedera:hedera@postgres/hedera
             REDIS_URL: redis://redis:6379
+            PYTHONDONTWRITEBYTECODE: 1
+            PYTHONUNBUFFERED: 1
         networks:
             - localdev
-        command: /bin/sh -c 'while ! nc -z postgres 5432; do sleep 1; done; while ! nc -z redis 6379; do sleep 1; done; python manage.py rqworker ${RQ_QUEUES:-default}'
+        command: /bin/sh -c 'while ! nc -z postgres 5432; do sleep 1; done; while ! nc -z redis 6379; do sleep 1; done; exec python manage.py rqworker ${RQ_QUEUES:-default}'
+        stop_signal: SIGINT
```

Comment:
This speeds up recycling containers. The default stop signal Docker sends to the container is SIGTERM, which Django never receives, so Docker waits ten seconds before killing the container with SIGKILL. Changing the stop signal to SIGINT will cause Docker to send Ctrl-C to the container; using `exec` in the script will cause that (last) script command to assume PID 1, and thereby allow Ctrl-C to be propagated to Django, resulting in a very fast graceful shutdown (usually less than a second).

This is particularly important for the `worker` container, as it doesn't auto-reload worker code.